Assets/Editor/E05_Controls.cs

```diff
@@ -77,6 +101,9 @@ void AddTask()
             {
                 m_TasksContainer.contentView.AddChild(CreateTask(m_TextField.text));
                 m_TextField.text = "";
+
+                // Give focus back to text field.
+                m_TasksContainer.focusController.SwitchFocus(m_TextField);
```

Comment:
Should we make Focus controller public? Isn't here an extension method to give focus to an object?